test/parallel/test-fs-realpath.js

```diff
@@ -200,7 +200,9 @@ function test_cyclic_link_protection(callback) {
     fs.symlinkSync(t[1], t[0], 'dir');
     unlink.push(t[0]);
   });
-  assert.throws(function() { fs.realpathSync(entry); });
+
+  const errorTooManySymLinks = new RegExp(`^Error: ELOOP: too many symbolic links encountered, stat '${entry}'$`);
```

Comment:
This line is too long. I thing the linter isn't happy with it. You can check this by running:

```js
make lint
```

I'm not sure if this will work on Windows as the `\` should be escaped in the regex. I suggest to move to a validating function instead of a regex. You can use something like this:

```js
(err) => {
  assert.strictEqual(
    err.message,
    `ELOOP: too many symbolic links encountered, stat '${entry}'`
  );
  return true;
}
```